clusters/build-clusters/01_cluster/Makefile

```diff
@@ -0,0 +1,17 @@
+#Reusing the pull secret from secret -n ci cluster-secrets-aws
+#It requires that oc-cli logs in api.ci.openshift.org
+pull_secret := $(shell oc get secret -n ci cluster-secrets-aws -o jsonpath='{.data.pull-secret}' | base64 -d)
```

Comment:
I thought we were going to use bitwarden for this?

Reply:
This is the pull secret from current CI cluster ... it is already from bitwarden.
I reuse it here.

Reply:
I meant -- what happens if we're installing this and there is no CI cluster? :)

Reply:
Hmm ... our bitwarden does not support multi-cluster yet.
In my opinion, bitwarden will create secrets for one cluster ... our secret-mirroring will do the mirror to other clusters, which is not implemented yet.
I think by then we can review how we handle the secrets.

Reply:
After discussion on [this](https://github.com/openshift/release/pull/5384#discussion_r335672949) tomorrow with @petr-muller, I should have a clearer plan for secrets.

Reply:
This is a general architecture question: should CI cluster be a central piece for secrets, or not? Should they flow from BW to CI cluster, and then be mirrored to other clusters, or should we invent something to put secrets from BW directly to the clusters they belong to? I believe both approaches have their merits.

For now, I think the BW -> CI -> build cluster is fine, as it reuses the building blocks we already have. It makes building build clusters dependent on CI cluster, but build clusters are not of much use without the CI one anyway...